common/persistence/class.SqlKvDriver.php

```diff
@@ -233,4 +243,20 @@ protected function gc()
         $statement = 'DELETE FROM kv_store WHERE kv_time > 0 AND kv_time <  ? ';
         return (bool)$this->sqlPersistence->exec($statement, [time()]);
     }
+
+    /**
+     * @inheritdoc
+     *
+     * @throws common_exception_InconsistentData
+     */
+    public function touchSchemas(SchemaCollection $schemaCollection)
```

Comment:
Are we creating here more than one schema? Why don't we name this method as createSchema anyway?
```suggestion
    public function createSchema(SchemaCollection $schemaCollection)
```